src/User32.Desktop/User32.cs

```diff
@@ -156,6 +177,12 @@ public static partial class User32
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool MessageBeep(MessageBeepType uType);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern MessageBoxResult MessageBox(IntPtr hWnd, string text, string caption, MessageBoxOptions options);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern int MessageBoxIndirect(ref MSGBOXPARAMS lpMsgBoxParams);
```

Comment:
> Couldn't make it blittable, could you help me pinpoint the problem? shouldn't I use char[] instead of char* on this case?

I'm not sure which case you're referring to (perhaps the comment is showing on the wrong diff line). But using `char*` with a `[Friendly(Array)]` attribute should generate a `char[]` parameter overload I think.